### PR TITLE
Add fetch call setup for all data sets

### LINKED ISSUE #5: Fetch Calls
Fetch data from this API [Local Server](https://github.com/turingschool-examples/fitlit-api)

GET Requests:

- [x]  User data `http://localhost:3001/api/v1/users`
- [x]  Sleep data `http://localhost:3001/api/v1/sleep`
- [x]  Activity data `http://localhost:3001/api/v1/activity`
- [x]  Hydration data `http://localhost:3001/api/v1/hydration`

### PR DESCRIPTION
**Pull request by Davidschlundtbodien**
# Description
Introduces fetch calls to the api inside main.js. This should be refactored as development continues. A console log is provided to ensure data was retrieved.

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactor feature (non-breaking change that modifies existing work)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?
- [ ] open index.html
- [x] console.log()
- [ ] dev tools
- [ ] npm test

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import UserRepository from './UserRepository'
+
+let userRepo, hydrationRepo, sleepRepo, activityRepo
+
+let fetchData = (dataType) => {
+  return fetch(`http://localhost:3001/api/v1/${dataType}`)
+    .then(res => {
+      return res.ok ? res.json() : console.log(`ERROR with ${dataType} path`)
```
Comment (joanafbrito) [inline src/main.js:8]: There is a interrogation point in this code, not sure about this. Would you mind explaining?

File: src/main.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+})
+
+let updateData = (data) => {
+  userRepo = new UserRepository(data[0].userData)
+  hydrationRepo = data[1].hydrationData
+  sleepRepo = data[2].sleepData
+  activityRepo = data[3].activityData
+}
+
+setTimeout(function(){ console.log(userRepo, hydrationRepo, sleepRepo, activityRepo); }, 2000);
```
Comment (joanafbrito) [inline src/main.js:24]: Not sure about why we need a console.log here. Would you mind explaining that?

**Review by joanafbrito — APPROVED** [review]
main.js line 8 - There is a interrogation point in this code, not sure about this. Would you mind explaining?
main.js line 24 - Not sure about why we need a console.log  Would you mind explaining that?

File: src/main.js (hunk 2 above)
Comment (Davidschlundtbodien) [reply src/main.js:24]: The console log is to check we are receiving our data correctly throughout development.

File: src/main.js (hunk 1 above)
Comment (Davidschlundtbodien) [reply src/main.js:8]: This is a Ternary to check if we have a 200 code : if not which fetch path has failed.

**Pull request merged**